### PR TITLE
worked on the Privacy Policy and Terms of Service page

### PR DESCRIPTION
**Pull request by omotoshoayomikun**

### CODE REVIEW

File: src/app/[locale]/privacy-policy/page.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { useTranslations } from 'next-intl';
+import styles from './PrivacyPolicy.module.css';
+function PrivacyPolicy() {
+
+    const lastUpdated = "3/1/2024";
```
Comment (swoocn) [inline src/app/[locale]/privacy-policy/page.tsx:6]: Can you re-align these constants?  
Also, please change the `lastUpdated` date to today's date.  Thanks!

File: src/app/[locale]/terms-of-service/page.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styles from './TermOfService.module.css';
+import { useTranslations } from 'next-intl';
+function TermsOfService() {
+
+  const t = useTranslations();
+  
+  const lastUpdated = "3/1/2024";
```
Comment (swoocn) [inline src/app/[locale]/terms-of-service/page.tsx:8]: Please change the `lastUpdated` date to today's date.  We'll eventually get this auto-versioned based on the release w/ CI/CD.

File: src/app/[locale]/privacy-policy/page.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      <li>{t('PRIVACY_POLICY.SECTIONS.CONTENT_7_1')}</li>
+    </ul>
+
+    <h2>8. {t('PRIVACY_POLICY.SECTIONS.HEADER_8')}</h2>
+    <ul>
+      <li>{t('PRIVACY_POLICY.SECTIONS.CONTENT_8_1')} &#64; <strong><a href={`mailto:${emailAddress}`}>Map of
+            Pi</a></strong>.</li>
+    </ul>
+</div>
+
```
Comment (swoocn) [inline src/app/[locale]/privacy-policy/page.tsx:92]: Can you remove one of the line feeds?  Looks like you have one too many.

**Comment by swoocn** [comment]
Hey @omotoshoayomikun - Great work again!
Is the decision to package with [locale] primarily driven by the need for translation in the files?  If so, would it make sense to package all the components under this package since they will all be subject for translation?  Perhaps there's another rationale behind this decision that I'm not aware of..?

Regardless, it seems like we should come up with a convention for restructuring the folders/ packages to enhance organization, possibly by categorizing them based on screens such as homescreen, buyer, seller, or through an alternative method.  What are your thoughts on this @omotoshoayomikun, @soleil00, @zolcsi?

File: src/app/[locale]/privacy-policy/page.tsx (hunk 1 above)
Comment (omotoshoayomikun) [reply src/app/[locale]/privacy-policy/page.tsx:6]: Do you mean updating the lastUpdated date to today's date, such as '05/05/2024,' or keeping track of the current date using the new Date() method?

**Comment by omotoshoayomikun** [comment]
The decision to package with [locale] is primarily driven by the need for translation in the files. However, initially, I chose not to package the components folder within it to maintain a clear distinction. This helps keep the folder focused on routes and other specific elements, aiding in the organization of files and making it easier to navigate the codebase.

I agree that coming up with a convention for restructuring the folders would be beneficial. It would enhance the organization and streamline development efforts. Additionally, if there's a preference for including the components folder inside the [locale] package, I'm open to that adjustment as well.

File: src/app/[locale]/privacy-policy/page.tsx (hunk 1 above)
Comment (swoocn) [reply src/app/[locale]/privacy-policy/page.tsx:6]: Just hardcoding it to 5/5/2024 is sufficient for now.  Thx!

**Comment by swoocn** [comment]
> The decision to package with [locale] is primarily driven by the need for translation in the files. However, initially, I chose not to package the components folder within it to maintain a clear distinction. This helps keep the folder focused on routes and other specific elements, aiding in the organization of files and making it easier to navigate the codebase.
> 
> I agree that coming up with a convention for restructuring the folders would be beneficial. It would enhance the organization and streamline development efforts. Additionally, if there's a preference for including the components folder inside the [locale] package, I'm open to that adjustment as well.

Here are my initial thoughts - 
1. I'm thinking we could rename the /messages folder to /translations and then put it under /src.
2. I think what we have in the **src/app/[locale]** is fine for now.  We should probably look towards organizing **src/components** with subfolders based on the screens i.e., seller, buyer, order, etc. 
We can refactor the project structure down the road as it continues to mature.  Thoughts?

File: src/app/[locale]/privacy-policy/page.tsx (hunk 3 above)
Comment (omotoshoayomikun) [reply src/app/[locale]/privacy-policy/page.tsx:92]: I don't understand what you mean

Comment (swoocn) [reply src/app/[locale]/privacy-policy/page.tsx:92]: Just remove the extra line feed @ L92.

**Review by swoocn — APPROVED** [review]
LGTM 👍

File: src/app/[locale]/privacy-policy/PrivacyPolicy.module.css
Comment (hollusam) [inline src/app/[locale]/privacy-policy/PrivacyPolicy.module.css:1]: I think SCSS was used before, and we can also try to minimise CSS by using tailwind where possible

**Pull request merged**